src/main/java/seedu/address/ui/ResultDisplay.java

```diff
@@ -38,4 +38,18 @@ public void setMessageFromUser(String messageFromUser) {
         );
     }
 
+    /**
+     * Scrolls up.
+     */
+    public void scrollUp() {
+        scrollPane.setVvalue(scrollPane.getVvalue() + 5);
```

Comment:
I tried pulling this PR, up on the left side does not seem to have an effect. Did you mean to do `- 5` instead?

Reply:
Great work on the right side panel though, that works well :)

Reply:
The problem was that when a new command is input, the scroll view will scroll to the last command. This includes the command that causes it to scroll up, so there's a paradox.

Reply:
So this will be fixed in the next iteration of the code?

Reply:
Yep!